docs/topics/whatsnew1520.md

```diff
@@ -0,0 +1,269 @@
+for string concatenation.
+
+To switch back to concatenation via [`StringBuilder.append()`](https://docs.oracle.com/javase/9/docs/api/java/lang/StringBuilder.html#append-java.lang.String-)
+used in previous versions, add the compiler option `-Xstring-concat=inline`.
+
+Learn how to add compiler options in [Gradle](gradle.md#compiler-options), [Maven](maven.md#specifying-compiler-options), and the [command-line compiler](compiler-reference.md#compiler-options).
+
+### Support for JSpecify nullness annotations
+
+The Kotlin compiler can read various types of [nullability annotations](java-interop.md#nullability-annotations) to pass
+nullability information from Java to Kotlin. Version 1.5.20 introduces support for the [JSpecify project](https://jspecify.dev/),
+which includes the standard unified set of Java nullness annotations.
+
+With JSpecify, you can provide more detailed nullability information to help Kotlin keep null-safety interoperating with
+Java. You can set default nullability for the declaration, package, or module scope, specify parametric nullability,
+and more. You can find more details about this in the [JSpecify design overview](https://jspecify.dev/design-overview.html).
+
+Here is the example of how Kotlin can handle JSpecify annotations:
+
+```java
+// JavaClass.java
+import org.jspecify.nullness.*;
+
+@NullMarked
+public class JavaClass {
+  public String notNullableString() { return ""; }
+  public @Nullable String nullableString() { return ""; }
+}
+```
+
+```kotlin
+// Test.kt
+fun kotlinFun() = with(JavaClass()) {
+  notNullableString().length // OK
+  nullableString().length    // Warning: receiver nullability mismatch
+}
+```
+
+In 1.5.20, all nullability mismatches according to the JSpecify-provided nullability information are reported as warnings.
+Use the `-Xjspecify-annotations=strict` and `-Xtype-enhancement-improvements-strict-mode` compiler options to enable
+strict mode (with error reporting) when working with JSpecify.
+
+[Learn more about null-safety and platform types](java-interop.md#null-safety-and-platform-types).
+
+### Support for calling Java’s Lombok-generated methods within modules that have Kotlin and Java code
+
+> The Lombok compiler plugin is [Experimental](components-stability.md).
+> It may be dropped or changed at any time. Use it only for evaluation purposes.
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+Kotlin 1.5.20 introduces an experimental Lombok compiler plugin <!-- [Lombok compiler plugin](lombok.md) -->. This plugin makes it possible to generate
+and use Java’s [Lombok](https://projectlombok.org/) declarations within modules that have Kotlin and Java code. Lombok
+annotations work only in Java sources and are ignored if you use them in Kotlin code.
+
+The plugin supports the following annotations:
+* Basic support for the `@With` and `Data` annotations
```

Comment:
@PetrakovichVictoria I think `@` is missing before the `Data`

Reply:
True, thanks!